### PR TITLE
Add dockerhub pull

### PR DESCRIPTION
**Pull request by vtomole**
I created an organization that hosts the Docker images: https://hub.docker.com/u/quantumlib. Please give me your Docker ID so I may add you.

### CODE REVIEW

**Comment by vtomole** [comment]
@zoltanegyed Are you saying that we should use the same Dockerfile for development  and non-development? Maybe have an if statement in the Dockerfile that sepcifies whether to `pip3 install cirq` or `git clone cirq` depending on if we want the image to be for development or not? If yes, i agree.

**Comment by zoltanegyed** [comment]
That is a good idea. We can also do that in a separate PR. I was just thinking that the instructions we give and the prebuilt images should be named the same (one name for dev, one for prod, but the same for prebuilt and self-built).

File: docs/development.md [hunk 1]
```diff
@@ -13,12 +13,10 @@ git clone git@github.com:quantumlib/cirq.git
 cd cirq
 ```
 
-To do your development in a Docker virtual machine, you can use dev_tools/Dockerfile:
+To do your development in a Docker image, you can build one with Cirq/dev_tools/Dockerfile or pull an existing image:
```
Comment (dabacon) [inline docs/development.md]: Cirq/dev_tools/Dockerfile doesn't exist?

Comment (vtomole) [reply docs/development.md]: Here: https://github.com/quantumlib/Cirq/blob/master/dev_tools/Dockerfile

Comment (vtomole) [reply docs/development.md]: @dabacon Ping :)

Comment (dabacon) [reply docs/development.md]: Ah thanks.  Could you make the doc link to that file?

Comment (vtomole) [reply docs/development.md]: Where should i put the link? On this file?  `docs/development.md`

**Comment by Strilanc** [comment]
Jeez, DockerHub is very bad at displaying our README.

**Comment by vtomole** [comment]
This is a sign to remove all the fancy stuff and make it a `README.txt`/s :laughing:

**Comment by vtomole** [comment]
@Strilanc Docker Hub doesn't render `.rst` which is what our README is. It renders `.md`.

**Pull request merged**